python/pyarrow/_array.pyx

```diff
@@ -64,6 +64,56 @@ cdef class DataType:
         else:
             raise TypeError('Invalid comparison')
 
+    def to_pandas_dtype(self):
+        """
+        Return the NumPy dtype that would be used for storing this
+        """
+        cdef Type type_id = self.type.id()
+
+        if type_id == _Type_NA:
+            # NaNs
+            return np.float64
+        elif type_id == _Type_BOOL:
+            return np.bool_
+        elif type_id == _Type_INT8:
+            return np.int8
+        elif type_id == _Type_INT16:
+            return np.int16
+        elif type_id == _Type_INT32:
+            return np.int32
+        elif type_id == _Type_INT64:
+            return np.int64
+        elif type_id == _Type_UINT8:
+            return np.uint8
+        elif type_id == _Type_UINT16:
+            return np.uint16
+        elif type_id == _Type_UINT32:
+            return np.uint32
+        elif type_id == _Type_UINT64:
+            return np.uint64
+        elif type_id == _Type_HALF_FLOAT:
+            return np.float16
+        elif type_id == _Type_FLOAT:
+            return np.float32
+        elif type_id == _Type_DOUBLE:
+            return np.float64
+        elif type_id == _Type_DATE32:
+            return np.dtype('datetime64[ns]')
+        elif type_id == _Type_DATE64:
+            return np.dtype('datetime64[ns]')
+        elif type_id == _Type_TIMESTAMP:
+            return np.dtype('datetime64[ns]')
+        elif type_id == _Type_BINARY:
+            return np.object_
+        elif type_id == _Type_FIXED_SIZE_BINARY:
+            return np.object_
+        elif type_id == _Type_STRING:
+            return np.object_
+        elif type_id == _Type_LIST:
+            return np.object_
+        else:
+            raise NotImplementedError(str(self))
```

Comment:
Yeah, this also isn't important for current work.  Just a side comment.  

Do you have an estimate about when a conda package (or other binary build artifact) will be available for development testing?